junifer/storage/base.py

```
@@ -51,9 +51,8 @@ def get_meta(self) -> Dict:
         }
         return meta
 
-    # TODO: is raising ValueError required?
     @abstractmethod
-    def validate(self, input_: List[str]) -> bool:
```

It still should be the same: `validate`. However, the storage has no output.

---

So it raises an error if `validate` fails?

---

From what I see, `validate` in `SQLiteFeatureStorage` returns a bool.

---

It should not. It should raise an error if it fails to validate the action.

---

Okay then I'll make the changes.